pkg/minikube/cluster/cluster.go

```diff
@@ -255,7 +262,9 @@ func DeleteHost(api libmachine.API) error {
 	}
 	// This is slow if SSH is not responding, but HyperV hangs otherwise, See issue #2914
 	if host.Driver.DriverName() == constants.DriverHyperv {
-		trySSHPowerOff(host)
+		if err := TrySSHPowerOff(host); err != nil {
+			return errors.Wrap(err, "Unable to power off minikube because the host was not found.")
```

Comment:
Make this into a logged warning: failure to power the VM off should not prevent it from being deleted.